packages/fast-components-foundation-react/README.md

```diff
@@ -94,9 +94,9 @@ Rendered result:
 <div class="my-class-name"></div>
 ```
 
-### `withSlot`
+### `withSlot` (Foundation only)
```

Comment:
I feel like this is ambiguous since Foundation is not a named export. Can you place a heading for the shared default and PureFoundation exports, then at the end another heading for the default export only methods?